pkg/filemanager/driver/oss/oss.go

```diff
@@ -439,7 +439,7 @@ func (handler *Driver) Source(ctx context.Context, e fs.Entity, args *driver.Get
 }
 
 func (handler *Driver) signSourceURL(ctx context.Context, path string, expire *time.Time, req *oss.GetObjectRequest, forceSign bool) (string, error) {
-	ttl := time.Duration(24) * time.Hour * 365 * 20
+	ttl := time.Duration(24) * time.Hour * 7 // V4 Sign 最大过期时间为7天
```

Comment:
Might need change in the doc as well: https://github.com/cloudreve/docs/blob/3b47e6f601e08724cf508f23b13cca753ca0ac65/zh/usage/storage/index.md?plain=1#L16